### PR TITLE
Fixed commented test from 11.05 PR

### PR DESCRIPTION
**Pull request by KamKoz92**
Fixed commented tests from #117 and old problematic test from #108

### CODE REVIEW

File: src/test/java/org/wildhamsters/battleships/fleet/FleetTests.java [hunk 1]
```diff
@@ -1,100 +1,99 @@
...
 import java.util.ArrayList;
 import java.util.List;
 
-import static org.testng.Assert.*;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
 
 public class FleetTests {
     @DataProvider(name = "fieldList-provider")
```
Comment (deetree) [inline src/test/java/org/wildhamsters/battleships/fleet/FleetTests.java:14]: fieldList-provider, delete the dash

File: src/test/java/org/wildhamsters/battleships/play/SingleShotTest.java [hunk 2]
```diff
@@ -138,13 +143,19 @@ private Fleet prepareFleet() {
...
-                                 Player player, int position) {
+            String currentTurnPlayer,
+            String currentTurnPlayerName,
+            String opponent,
+            Integer... sunkShipPositions) {
+
+        // List<Integer> sunkenShipList = (player.getSunkShipPositions(position).size()
+        // > 1)
+        // ? player.getSunkShipPositions(position)
+        // : null;
```
Comment (deetree) [inline src/test/java/org/wildhamsters/battleships/play/SingleShotTest.java:154]: delete the comment

**Review by deetree — CHANGES_REQUESTED** [review]
Fix the issues then it'll be merged.

File: src/test/java/org/wildhamsters/battleships/play/SingleShotTest.java (hunk 2 above)
Comment (KamKoz92) [reply src/test/java/org/wildhamsters/battleships/play/SingleShotTest.java:154]: Removed.

File: src/test/java/org/wildhamsters/battleships/fleet/FleetTests.java (hunk 1 above)
Comment (KamKoz92) [reply src/test/java/org/wildhamsters/battleships/fleet/FleetTests.java:14]: Fixed name for provider.

**Review by deetree — APPROVED** [review]
👍🏻

**Pull request merged**